Library/Homebrew/download_strategy.rb

```diff
@@ -271,6 +272,16 @@ def initialize(name, resource)
   end
 
   def fetch
+    if ARGV.force_domain?
+      unless (ENV["HOMEBREW_ARTIFACT_DOMAIN"].nil?)
+        if (!(@url.include? ENV["HOMEBREW_ARTIFACT_DOMAIN"]) && !(@url.include? "https://homebrew.bintray.com"))
```

Comment:
the `bintray.com` should use a variable rather than being hardcoded. Also, why do you exclude results from there?